src/components/Footer.tsx

```diff
@@ -1,38 +1,33 @@
-import { FaGithub, FaTwitter, FaLinkedinIn, FaInstagram } from 'react-icons/fa/index';
+import {
+  FaGithub,
+  FaTwitter,
+  FaLinkedinIn,
+  FaInstagram,
+} from 'react-icons/fa/index';
 
 const Footer = ({ isDark }: { isDark?: boolean }) => (
-  <footer
-    className={`text-sm shadow-inner-xs ${
-      isDark ? 'bg-slate-900' : 'bg-slate-100'
```

Comment:
Can we use the same colors that were being used earlier for the dark mode of footer? I think it already looked good.

Also, since you have removed usage of `isDark` in the component, it's also better to remove it from component props.